lib/internal/process/main_thread_only.js

```diff
@@ -98,6 +104,18 @@ function setupPosixMethods(_initgroups, _setegid, _seteuid,
     }
   };
 
+  function validateInc(inc) {
+    const tInc = typeof inc;
```

Comment:
Can you inline `typeof inc` here? The engine is generally better at recognizing that pattern and can eliminate the temporary string you’d be creating here

Reply:
Thanks for the deep insights. It is fixed.